### ISSUE TITLE
Position Order get to 0 after Gnome Restart?

### ISSUE THREAD

**Issue by rizkytahara** [caasiu/gnome-shell-extension-KeysIndicator#6]
Hi,

If set the position order to have value other than 0, it always back again to 0 after I restart Gnome using `ALT + F2` and `r`

How can I set the value to be persistent?

Fedora 25
Gnome 3.22

**Comment by caasiu** [comment]
That's weird! I set the default value of position order to 1. 

You can edit `schemas/org.gnome.shell.extensions.keysindicator.gschema.xml` file under your local `keys-indicator@caasiu.github.com` folder, change <kbd>positon-order</kbd> default value `1` to what you want.

let me know if it works

**Comment by rizkytahara** [comment]
nope.. it work when i apply the value, but changed again to 0 after gnome shell restart

**Comment by caasiu** [comment]
I forget telling you to run `glib-compile-schemas ./` under schemas folder after change the xml file.

Could you try it again?

**Comment by rizkytahara** [comment]
still not working..
in the file is already set to value other than 0, but the position still back to the most right..
at the dconf, the value get back to 0

**Comment by caasiu** [comment]
Try install latest version, it should works.
`0` means very left in the corner.
But if another extension enabled after it, its position will change.

**Comment by caasiu** [comment]
Hi, @rizkytahara 
Does the latest version solve the issue?

**Comment by rizkytahara** [comment]
not really...
- now the number count from the left to the right? increasing the position, goes to the right?
- when i set to 1, restart the shell.. the position on the top bar goes right +1.. but in the config still 1...
